include/deal.II/lac/sparse_matrix_ez.h

```diff
@@ -86,7 +86,7 @@ template<typename number> class FullMatrix;
  * If the rows are expected to be filled more or less from first to last,
  * using a @p default_row_length of zero may not be such a bad idea.
  *
- * The name of this matrix is in reverence to a publication of the Internal
+ * The name of this matrix is in reference to a publication of the Internal
```

Comment:
Ha, that's actually quite funny. Both the old and new spelling make sense, I suppose :-)